hubcap/records.py

```diff
@@ -176,7 +177,7 @@ def make_spec(self, org, repo, package_name, packages, version):
             "version": version,
             "published_at": "1970-01-01T00:00:00.000000+00:00",
             "packages": packages,
-            "works_with": [],
+            "works_with": require_dbt_version,
```

Comment:
`require-dbt-version` can be a string or a list ([docs](https://docs.getdbt.com/reference/project-configs/require-dbt-version)), so the resulting value in the Hub API can be a string or a list.

I added [logic to handle this](https://github.com/dbt-labs/dbt-core/pull/5651/files#diff-56648fa17b1711068e02ac1031a915b3e2dbc667a973b057c2c711e69054c134R136-R139) within `dbt-core`, rather than having that logic live here (= turning `string` into `[string]`).